### PR TITLE
Subscribe

### PR DESCRIPTION
**Pull request by manggalang**
within responsive

### CODE REVIEW

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 1]
```diff
@@ -17,6 +17,18 @@
 		line-height: 2;
 		font-weight: normal;
 		margin: 0;
+
+		@media only screen and (max-width: 900px){
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_subscribe.scss]: try to change the font when it's hit 768px and 480px

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 2]
```diff
@@ -17,6 +17,18 @@
 		line-height: 2;
 		font-weight: normal;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_subscribe.scss]: change it to 400

File: app/assets/stylesheets/pages/home/_subscribe.scss [hunk 3]
```diff
@@ -38,5 +50,17 @@
 	    	background-color: #fff;
 	    	color: #484848;
 	    }
+
+	    @media only screen and (max-width: 900px){
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_subscribe.scss]: try to change the font when it's hit 768px and 480px

File: app/assets/stylesheets/pages/home/_subscribe.scss (hunk 1 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_subscribe.scss]: okay, already to change

File: app/assets/stylesheets/pages/home/_subscribe.scss (hunk 3 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_subscribe.scss]: okay, already to change this

File: app/assets/stylesheets/pages/home/_subscribe.scss (hunk 2 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_subscribe.scss]: sudah

**Pull request merged**